src/app/tests/utils/utils.py

```diff
@@ -0,0 +1,26 @@
+import random
+import string
+
+
+def random_string() -> str:
+    return "".join(random.choices(string.ascii_lowercase, k=32))
+
+
+def random_int(negative=True) -> int:
+    if negative:
+        return random.randint(-5000, 5000)
+    else:
+        return random.randint(0, 5000)
+
+
+def random_float(negative=True) -> float:
+    if negative:
+        return random.uniform(-5.0, 5.0)
+    else:
+        return random.uniform(0.0, 5.0)
```

Comment:
```suggestion
    return random.uniform(-5.0 if negative else 0.0, 5.0)
```
Or does this look weird?